core/trino-main/src/main/java/io/trino/operator/join/DefaultPagesHash.java

```diff
@@ -257,6 +259,12 @@ public void appendTo(long position, PageBuilder pageBuilder, int outputChannelOf
         pagesHashStrategy.appendTo(blockIndex, blockPosition, pageBuilder, outputChannelOffset);
     }
 
+    @Override
+    public boolean isMappingUnique()
```

Comment:
not sure if this is part of this PR but would be good to have this available in OperatorStats